redcycle/views/Signup.py

```diff
@@ -0,0 +1,17 @@
+from django.contrib.auth import login, authenticate
+from django.shortcuts import render, redirect
+from ..forms.CustomUserCreationForm import *
+
+def Signup(request):
+    if request.method == 'POST':
+        form = CustomUserCreationForm(request.POST)
+        if form.is_valid():
+            form.save()
+            email = form.cleaned_data.get('email')
+            raw_password = form.cleaned_data.get('password1')
+            user = authenticate(email=email, password=raw_password)
+            login(request, user)
+            return redirect('/redcycle/post-creator')
+    else:
+        form = CustomUserCreationForm()
+    return render(request, 'registration/signup.html', {'form': form})
```

Comment:
Custom signup view. Redirects to `/redcycle/post-creator` after signup is complete